jsk_2014_picking_challenge/BinContents[] array
